<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_proveedores</class>
 <widget class="QWidget" name="form_proveedores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proveedores</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(3, 3, 3);
background-color: rgb(44, 43, 58);
font: 10pt &quot;Liberation Sans&quot;;
color: rgb(252, 252, 252);
selection-background-color: rgb(155, 155, 185);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>CUIT / CUIL: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_cuit">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Cliente:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbx_cliente"/>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Razón Social: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_razon_social">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QPushButton" name="btn_buscar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(64, 63, 78);</string>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="btn_guardar">
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">QTabWidget{
background-color: rgb(123, 121, 143);
}
QLineEdit {
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}
QComboBox{
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Proveedores</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Nombre :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lne_nombre">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="horizontalSpacer_11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Razón Social :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lne_razon_social_proveedor">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>CUIT / CUIL :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lne_cuit_proveedor">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Condición Fte. IVA:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbx_iva">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <item>
              <property name="text">
               <string>Resp. Monotributo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Resp. Inscripto</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Exento</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>No Alcanzado</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Consumidor Final</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>95</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>95</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Referencia: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lne_ref">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lne_cuit</tabstop>
  <tabstop>lne_razon_social</tabstop>
  <tabstop>btn_buscar</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>lne_nombre</tabstop>
  <tabstop>lne_razon_social_proveedor</tabstop>
  <tabstop>lne_cuit_proveedor</tabstop>
  <tabstop>cbx_iva</tabstop>
  <tabstop>lne_ref</tabstop>
  <tabstop>btn_guardar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
